lib/CL/devices/remote/communication.c

```diff
@@ -2082,10 +2083,15 @@ pocl_network_create_buffer (remote_device_data_t *ddata, cl_mem mem,
 
   nc.request.m.create_buffer.flags = mem->flags;
   nc.request.m.create_buffer.size = mem->size;
-  // see https://www.gnu.org/software/c-intro-and-ref/manual/html_node/Pointer_002dInteger-Conversion.html
-  // for reason behind the double cast
+  /* See https://www.gnu.org/software/c-intro-and-ref/manual/html_node/
```

Comment:
This breaks the link, should it be surrounded in `<>` to keep it intact during formatting?